src/components/App/App.js

```diff
@@ -168,22 +166,12 @@ class App extends React.Component {
           />
           )} />
 
-        <Route user={user} path='/create-workspace' render={() => (
-          <WorkSpaceCreate
-            user={user}
-            placeId={this.state.placeId}
-            placeData={this.state.placeData}
-            workspaceLocation={this.state.poiLocation}
-            setApp={this.setState.bind(this)}
-          />
-          )} />
-
```

Comment:
This code is outdated. No longer using this route or `WorkSpaceCreate` component